### PR TITLE
[IOTNGGW-241]Add ims tail log support with fluentbit

### PR DESCRIPTION
**Pull request by TapaniPuhakka**

### CODE REVIEW

**Review by kivaisan — DISMISSED** [review]

**Review by JanneKiiskila — APPROVED** [review]

File: recipes-extended/fluentbit/fluentbit_%.bbappend [hunk 1]
```diff
@@ -6,6 +6,9 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 FB_PKG_NAME = "td-agent-bit"
 FB_CONF_FILES_LOCATION = "/etc/${FB_PKG_NAME}"
 
+#Set this to "1" in local.conf to enable IMS logs via fluentbit to pelion
```
Comment (JanneKiiskila) [inline recipes-extended/fluentbit/fluentbit_%.bbappend:9]: # space S

Comment (JanneKiiskila) [reply recipes-extended/fluentbit/fluentbit_%.bbappend:9]:
```suggestion
# Set this to "1" in local.conf to enable IMS logs via fluentbit to pelion
```

**Comment by JanneKiiskila** [comment]
Ready to merge? @TapaniPuhakka  @x85446  @petedyerarm

**Comment by JanneKiiskila** [comment]
@x85446  - review and merge, please?

File: recipes-extended/fluentbit/files/td-agent-bit-ims-tail-input.conf [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+[INPUT]
+    # use tail to read IMS logs
+    Name        tail
+    Tag  ${ima_tag}
+    # Just read all ascii measurements from IMA
+    Path        /sys/kernel/security/ima/ascii_runtime_measurements
+    # Note you can't add extra fields or Pelion cloud will not accept, eg do not enable below line
+    # Path_Key filename
+    
+[FILTER]
```
Comment (JanneKiiskila) [inline recipes-extended/fluentbit/files/td-agent-bit-ims-tail-input.conf:13]: This filter could be removed? @AriParkkila  @TapaniPuhakka

**Review by x85446 — APPROVED** [review]

**Pull request merged**